CISS/transforms/CISS_Injury_Cleaning.py

```diff
@@ -19,12 +21,6 @@
 output_dataframes = []
 output_filename = clean_directory + '/INJURY_CLEANED.csv'
 
-# Initialize the maps to convert AIS and body region from numbers to their descriptors
-ais_map = {1: 'Minor Injury', 2: 'Moderate Injury', 3: 'Serious Injury', 4: 'Severe Injury', 5: 'Critical Injury',
-           6: 'Maximum Injury', 9: 'Injured, Unknown Severity'}
-region_map = {0: 'Other Trauma', 1: 'Head', 2: 'Face', 3: 'Neck', 4: 'Thorax', 5: 'Abdomen', 6: 'Spine',
-              7: 'Upper Extremity', 8: 'Lower Extremity', 9: 'Unspecified'}
-
 
 def clean_ciss():
```

Comment:
Might be out of scope but could probably refactor to use same logic as clean_patient_files()

Reply:
Will also add this shortly